src/modules/Library/features/Applet/Applet.tsx

```diff
@@ -92,9 +118,11 @@ export const Applet = ({
               {t('viewDetails')}
             </Button>
             <Button
+              disabled={addToBuilderDisabled}
```

Comment:
`addToBuilderDisabled` supposed to be a boolean, maybe rename it to `isAddToBuilderDisabled` or  `shouldAddToBuilder` and use opposite value for it `!shouldAddToBuilder`?